authors/apps/comments/api_views.py

```diff
@@ -0,0 +1,64 @@
+from rest_framework import generics
+from rest_framework import permissions
+from authors.apps.articles.models import Article
+from rest_framework import response
+from .models import Comment
+from rest_framework import status
+from .serializers import CommentSerializer
+from .renderers import CommentsRenderer
+from authors.apps.authentication.models import User
+from django.db.models import Q
+from authors.apps.core import authority
+
+
+class CommentList(generics.ListCreateAPIView):
+
+    queryset = Comment.objects.all()
+    serializer_class = CommentSerializer
+    renderer_classes = (CommentsRenderer,)
+    lookup_field = "slug"
+    permission_classes = (permissions.IsAuthenticatedOrReadOnly,)
+
+    def post(self, request, *args, **kwargs):
+        comment = request.data.get("comment", {})
+        serializer = CommentList.serializer_class(data=comment)
+        serializer.is_valid(raise_exception=True)
+        user = request.user
+        slug = kwargs["slug"]
+        articles = Article.objects.filter(slug=slug).first()
```

Comment:
Please add a check when the article doesn't exist

Reply:
Thanks @jeanjoe , I have implemented that check.